dist/redhat_access_angular_ui_common.js

```diff
@@ -713,6 +713,16 @@ angular.module('RedhatAccess.header').directive('rhaHeader', function () {
 });
 
 'use strict';
+/*jshint unused:vars */
```

Comment:
@ardasgup That is under TODO item(i guess). Right now , we need to push this common file otherwise the changes would not be into effect.